src/app/parcels/parcelsTable/ParcelsPage.test.tsx

```diff
@@ -0,0 +1,236 @@
+import { processEventsDetails } from "@/app/parcels/getExpandedParcelDetails";
+import {
+    familyCountToFamilyCategory,
+    formatAddressFromClientDetails,
+    formatBreakdownOfChildrenFromFamilyDetails,
+    formatHouseholdFromFamilyDetails,
+} from "@/app/clients/getExpandedClientDetails";
+import { formatDatetimeAsDate } from "@/common/format";
+import { DbParcelRow } from "@/databaseUtils";
+import convertParcelDbtoParcelRow, {
+    processLastStatus,
+} from "@/app/parcels/parcelsTable/convertParcelDBtoParcelRow";
+import { CongestionChargeDetails, ParcelsTableRow } from "@/app/parcels/parcelsTable/types";
+import { expect, it } from "@jest/globals";
+import "@testing-library/jest-dom/jest-globals";
+
+jest.mock("@/supabaseClient", () => {
+    return { default: jest.fn() };
+});
+
+const logID = "a2adb0ba-873e-506b-abd1-8cd1782923c8";
+jest.mock("@/logger/logger", () => ({
+    logErrorReturnLogId: jest.fn(() => Promise.resolve(logID)),
+}));
+
+const sampleProcessingData: DbParcelRow[] = [
+    {
+        parcel_id: "PRIMARY_KEY",
+        collection_centre_name: "COLLECTION_CENTRE",
+        collection_centre_acronym: "CC",
+        collection_datetime: "2023-08-04T13:30:00+00:00",
+        packing_date: "2023-08-04T13:30:00+00:00",
+        packing_slot_name: "AM",
+        packing_slot_order: 1,
+        voucher_number: "VOUCHER_1",
+        client_id: "PRIMARY_KEY2",
+        client_full_name: "CLIENT_NAME",
+        client_address_postcode: "SW1A 2AA",
+        client_phone_number: "08642 123",
+        client_flagged_for_attention: false,
+        client_signposting_call_required: true,
+        family_count: 3,
+        is_delivery: false,
+        last_status_event_name: "LAST_EVENT",
+        last_status_event_data: "SOME_RELATED_DATA",
+        last_status_timestamp: "2023-08-04T13:30:00+00:00",
+        last_status_workflow_order: 1,
+        created_at: "2023-12-31T12:00:00+00:00",
+        client_is_active: true,
+    },
+];
+
+const sampleCongestionChargeData: CongestionChargeDetails[] = [
+    {
+        postcode: "SW1A 2AA",
+        congestionCharge: true,
+    },
+];
+
+describe("Parcels Page", () => {
+    describe("Backend Processing for Table Data", () => {
+        it("Fields are set correctly", async () => {
+            const { parcelTableRows } = await convertParcelDbtoParcelRow(
+                sampleProcessingData,
+                sampleCongestionChargeData
+            );
```

Comment:
the 'const {parcelTableRows}' should be typed. use 'as Type' after the converParcelDbtoParcelRow call for the relevant type